wp-content/themes/core/components/pagination/comments/Comments_Pagination_Controller.php

```diff
@@ -0,0 +1,100 @@
+<?php
+declare( strict_types=1 );
+
+namespace Tribe\Project\Templates\Components\pagination\comments;
+
+use Tribe\Libs\Utils\Markup_Utils;
+use Tribe\Project\Templates\Components\Abstract_Controller;
+
+class Comments_Pagination_Controller extends Abstract_Controller {
+	public const CLASSES = 'classes';
+	public const ATTRS   = 'attrs';
+	public const PAGED   = 'paged';
+
+	private array $classes;
```

Comment:
are we doing typed properties now?

Reply:
I don't know. I kind of like them. That's one of the things we can discuss here. Thoughts?